phoenix5-spark/src/main/java/org/apache/phoenix/spark/datasource/v2/PhoenixDataSource.java

```diff
@@ -42,6 +45,7 @@ public class PhoenixDataSource  implements DataSourceV2,  ReadSupport, WriteSupp
     private static final Logger logger = LoggerFactory.getLogger(PhoenixDataSource.class);
     public static final String SKIP_NORMALIZING_IDENTIFIER = "skipNormalizingIdentifier";
     public static final String ZOOKEEPER_URL = "zkUrl";
```

Comment:
You added a Deprecated tag to the ZOOKEEPER_URL in phoenix5-spark3/src/main/java/org/apache/phoenix/spark/sql/connector/PhoenixDataSource.java 
why not here too?

Reply:
Corrected.